src/v2/view-builder/views/IdentifierView.js

```diff
@@ -113,15 +113,23 @@ const Body = BaseForm.extend({
     const passwordExplainLabeli18nKey = 'primaryauth.password.tooltip';
 
     const newSchemas = schemas.map(schema => {
-      const newSchema = { ...schema };
+      let newSchema = { ...schema };
 
```

Comment:
<img width="1333" alt="Screen Shot 2021-04-21 at 6 56 20 PM" src="https://user-images.githubusercontent.com/17267130/115644340-46873f80-a2d3-11eb-97ee-4d219c2ae810.png">


The inputs already have a data-se am I missing something

Reply:
That's not the same spot where I need the selector. 
My use case is I want to test the label and sub-labels for username and password and they're currently not using any descriptive classnames or attrs, so I would need to base my tests off of their position in the DOM. For example: 
```
usernameContainer = cssSelector([data-se="o-form-label"]).get(0)
passwordContainer = cssSelector([data-se="o-form-label"]).get(1)
```

Reply:
what tests need those selector?
what we usually do is locate the `input` and look up for its label (`parent -> sibling`)

Reply:
https://github.com/okta/okta-core/pull/52020

Reply:
In v1, the form had more descriptive attributes to facilitate testing 
![image](https://user-images.githubusercontent.com/71431120/115763430-4f225900-a359-11eb-8009-d6faa6220ac5.png)

Reply:
ahh I see. We can pass inputId attribute in here so all inputs get a selector for the label :) 

https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/uiSchemaTransformer.js#L61

eg) http://courage.oktaskylab.com/gist/vcZejrtNR

/cc @haishengwu-okta  if you have any other suggestions.

Reply:
I decided to use `data-se` instead of `inputId` because then I can also grab the sub-label (.o-form-explain)

Reply:
If it's merely for testing, I think we could enhance `FormComponent.java` by adding method like `getLabelForField`.
Otherwise, @magizh-okta 's suggestion is nice.

Reply:
The sub/explain label I am also testing is not a label, so inputId would not affect those elements.
> I decided to use `data-se` instead of `inputId` because then I can also grab the sub-label (.o-form-explain)

Reply:
@haishengwu-okta  I guess this is for selenium tests and getting outer element in selenium is tricky. @sherizada-okta  can we add the data-se={label+fieldName} in the UISchemaTransfer so all inputs gets this goodness.
We can create a follow up too if u want to get this before cF.

Reply:
I want to get this in before CF because it's blocking my monolith changes, and adding them to all elements might break existing selenium tests so that would need a monolith CI run.

Reply:
oh, yeah, it would need to utilize xpath.
the changes are too specific like v1 we specially add ID for username and password.

Reply:
I think adding a data-se that includes the form name to the default ui schema config is a good idea, but it'll require
- applying it to schemas that are not extending from the uiSchemaDefaultConfig in uiSchemaTransformer
- fix breaking testcafe tests
- check against monolith se tests

Reply:
Please refactoring for general solution as follow up and get rid of this one off.

Reply:
https://oktainc.atlassian.net/browse/OKTA-389414